network/meson.build

```diff
@@ -1,22 +1,41 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 if conf.get('ENABLE_NETWORKD') == 1
-        install_data('80-6rd-tunnel.network',
-                     '80-container-host0.network',
-                     '80-container-vb.network',
-                     '80-container-ve.network',
-                     '80-container-vz.network',
-                     '80-ethernet.network.example',
-                     '80-vm-vt.network',
-                     '80-wifi-adhoc.network',
-                     '80-wifi-ap.network.example',
-                     '80-wifi-station.network.example',
-                     install_dir : networkdir)
+        install_data(
+                '80-6rd-tunnel.network',
+                '80-container-host0.network',
+                '80-container-vb.network',
+                '80-container-ve.network',
+                '80-container-vz.network',
+                '80-vm-vt.network',
+                '80-wifi-adhoc.network',
+                install_dir : networkdir)
 
-        if install_sysconfdir
-                install_emptydir(sysconfdir / 'systemd/network')
+        example_network_files = files(
```

Comment:
Adding `files()` here seems to install those files into a another `network` subdirectory ie `/usr/lib/systemd/network/network` . Not sure that was intended.

Reply:
Yeah, fix is already waiting in #29930.